test/integration/features/cmd-addons.feature

```diff
@@ -82,6 +82,7 @@ Feature: Addons command and its subcommands
   Scenario: Starting Minishift with anyuid and xpaas add-ons enabled
   The addons are being applied in correct order.
     Given Minishift has state "Does Not Exist"
+      And image caching is disabled
      When executing "minishift start" succeeds
```

Comment:
IMO some kind of start log validation required to confirm that it uses the cached image.  
For example: 
```When executing "minishift start" succeeds
    Then stdout should contain/match "caching image string"
```

Reply:
@amitkrout I think it is not going to cache neither use the cached images as the image caching is disabled.